aspnetcore/toc.yml

```diff
@@ -0,0 +1,807 @@
+            - name: Textarea
+              href: mvc/views/working-with-forms.md#the-textarea-tag-helper
+            - name: Validation Message
+              href: mvc/views/working-with-forms.md#the-validation-message-tag-helper
+            - name: Validation Summary
+              href: mvc/views/working-with-forms.md#the-validation-summary-tag-helper
+    - name: Layout
+      href: uid:mvc/views/layout
+    - name: Static files
+      href: uid:fundamentals/static-files
+    - name: Model binding
+      href: uid:mvc/models/model-binding
+    - name: Model validation
+      href: uid:mvc/models/validation
+    - name: Razor syntax
+      href: uid:mvc/views/razor
+    - name: Advanced
+      items: 
+        - name: View components
+          href: uid:mvc/views/view-components
+        - name: View compilation
+          href: uid:mvc/views/view-compilation
+        - name: App model
+          href: uid:mvc/controllers/application-model
+        - name: Filters
+          href: uid:mvc/controllers/filters
+        - name: Areas
+          href: uid:mvc/controllers/areas
+        - name: App parts
+          href: uid:mvc/extensibility/app-parts
+        - name: Custom model binding
+          href: uid:mvc/advanced/custom-model-binding
+        - name: Compatibility version
+          href: uid:mvc/compatibility-version
+- name: Web API apps
+  items: 
+    - name: Overview
+      href: uid:web-api/index
+    - name: Tutorials
+      items: 
+        - name: Create a web API
+          href: uid:tutorials/first-web-api
+        - name: Web API with MongoDB
+          href: uid:tutorials/first-mongo-app
+    - name: Swagger / OpenAPI
+      items: 
+        - name: Overview
+          href: uid:tutorials/web-api-help-pages-using-swagger
+        - name: Get started with Swashbuckle
+          href: uid:tutorials/get-started-with-swashbuckle
+        - name: Get started with NSwag
+          href: uid:tutorials/get-started-with-nswag
+    - name: Action return types
+      href: uid:web-api/action-return-types
+    - name: Format response data
+      href: uid:web-api/advanced/formatting
+    - name: Custom formatters
+      href: uid:web-api/advanced/custom-formatters
+    - name: Analyzers
+      href: uid:web-api/advanced/analyzers
+    - name: Conventions
+      href: uid:web-api/advanced/conventions
+- name: Real-time apps
+  items: 
+    - name: SignalR overview
+      href: uid:signalr/introduction
+    - name: Supported platforms
+      href: uid:signalr/supported-platforms
+    - name: Tutorials
+      items: 
+        - name: SignalR with JavaScript
+          href: uid:tutorials/signalr
+        - name: SignalR with TypeScript
+          href: uid:tutorials/signalr-typescript-webpack
+    - name: Samples
+      href: https://github.com/aspnet/SignalR-samples
+    - name: Server concepts
+      items: 
+        - name: Hubs
+          href: uid:signalr/hubs
+        - name: HubContext
```

Comment:
I suggest using a different name. The name `HubContext` is meaningless to those who've never used the feature.